### PR TITLE
echo server config.

### PR DESCRIPTION
**Pull request by AntonSkiba**

### CODE REVIEW

File: src/front-end/app.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+const app = express();
+app.set("view engine", "ejs");
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.get("/", (req, res) => {
+  console.log("nodejs server");
+  res.render("index");
+});
+
+app.get("/pythonServer", (req, res) => {
```
Comment (IlyasYOY) [inline src/front-end/app.js]: You'd better replace it with `/api`

File: src/front-end/app.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  console.log("python server");
+  let xhttp = new XMLHttpRequest();
```
Comment (IlyasYOY) [inline src/front-end/app.js]: replace with fetch

File: src/front-end/views/index.ejs [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  <head>
+    <meta charset="utf-8">
+    <title>INDEX</title>
+  </head>
+  <body>
+    <button type="button" name="button" onclick="getReq()">Python server</button>
+    <div id="pam"><div>
+    <script type="text/javascript">
+    function getReq() {
+      let xhttp = new XMLHttpRequest();
```
Comment (IlyasYOY) [inline src/front-end/views/index.ejs]: replace with fetch

File: src/front-end/app.js (hunk 1 above)
Comment (AntonSkiba) [reply src/front-end/app.js]: fixed

File: src/front-end/app.js (hunk 2 above)
Comment (AntonSkiba) [reply src/front-end/app.js]: fixed

File: src/front-end/views/index.ejs (hunk 3 above)
Comment (AntonSkiba) [reply src/front-end/views/index.ejs]: fixed

**Pull request merged**